### PR TITLE
Simple documentation update to use capital letters for first argument

### PR DESCRIPTION
**Pull request by flyaflya**

### CODE REVIEW

File: docs/src/generated/visual.jl [hunk 1]
```diff
@@ -3,7 +3,8 @@
 # The function `visual` can be used to give data-independent visual information about the plot
 # (plotting function or attributes).
 
-# The available plotting functions are documetned [here](http://makie.juliaplots.org/dev/examples/plotting_functions/).
+# The available plotting functions are documented [here](http://makie.juliaplots.org/dev/examples/plotting_functions/).
+# Capitalize the first-letter of the function to use it as an argument to `visual` (e.g. `visual(Scatter)`).
```
Comment (piever) [inline docs/src/generated/visual.jl:7]: Maybe it would be best to mention that it's actually CamelCase, so for example `barplot` becomes `BarPlot`.

**Comment by piever** [comment]
Sorry for the slow reply! I've left a small comment.

**Review by flyaflya — COMMENTED** [review]
made slight mod as requested.  thx!

File: docs/src/generated/visual.jl (hunk 1 above)
Comment (flyaflya) [reply docs/src/generated/visual.jl]: good point :-)

**Comment by piever** [comment]
Great, thank you

**Pull request merged**